demo/demo.md

```diff
@@ -383,3 +383,43 @@ The `auro-icon` element comes with some pre-defined opinions, but these are easi
   <!-- AURO-GENERATED-CONTENT:END -->
 
 </auro-accordion>
+
+## Setting focus programmatically
+
+The following example illustrates a use case where a user may want to set a user's focus programmatically. I.e. through another click event.
+
+<auro-button onclick="getFocus()">click me for focus</auro-button>
+
+
+<!-- <div class="exampleWrapper">
+  <auro-hyperlink id="getFocus" href="https://www.alaskaair.com">Welcome to Alaska Airlines</auro-hyperlink>
+</div> -->
```

Comment:
Can we remove this or is it part of the auto-generation process?

Reply:
Part of the automated process.